nat/src/stateless/test.rs

```diff
@@ -26,12 +26,15 @@ mod tests {
 
```

Comment:
No shuttle tests?  We probably want a test that does translation from multiple threads since we grab a lock in this code.

Reply:
I don't have one at the moment. I referenced this comment on #793.